templates/servicefabric.tmpl

```diff
@@ -0,0 +1,143 @@
+[backends]
+    {{$groupedServiceMap := getServicesWithLabelValueMap .Services "backend.group.name"}}
+    {{range $aggName, $aggServices := $groupedServiceMap }}
+      [backends."{{$aggName}}"]
+      {{range $service := $aggServices}}
+        {{range $partition := $service.Partitions}}
+          {{range $instance := $partition.Instances}}
+            {{if and (isHealthy $instance) (hasHTTPEndpoint $instance)}} 
```

Comment:
I think this kind of test/filter should be in the provider itself

Reply:
Sure, makes more sense, I'll move this into the provider